zebra/interface.h

```diff
@@ -343,9 +343,9 @@ struct zebra_if {
 	/* Note: If additional fields become necessary, this can be modified to
 	 * be a pointer to a dynamically allocd struct.
 	 */
-	struct zebra_l2info_brslave brslave_info;
+	struct zebra_l2info_brmember brsecondary_info;
```

Comment:
Inconsistent, both replacements should be the same...

Reply:
Thanks for catching this!